src/main/scala/zio/sqs/SqsStream.scala

```diff
@@ -15,14 +15,16 @@ object SqsStream {
     settings: SqsStreamSettings = SqsStreamSettings()
   ): Stream[Throwable, Message] = {
 
-    val request = ReceiveMessageRequest.builder
+    val builder = ReceiveMessageRequest.builder
       .queueUrl(queueUrl)
       .attributeNamesWithStrings(settings.attributeNames.asJava)
       .messageAttributeNames(settings.messageAttributeNames.asJava)
       .maxNumberOfMessages(settings.maxNumberOfMessages)
-      .visibilityTimeout(settings.visibilityTimeout)
-      .waitTimeSeconds(settings.waitTimeSeconds)
-      .build
+
+    settings.visibilityTimeout.foreach(builder.visibilityTimeout(_))
```

Comment:
Is this working? Doesn't each method return a new builder that should be used?

Reply:
As far as I can tell, it does work -- for example,
```scala
val builder = ReceiveMessageRequest.builder

println(builder.build.visibilityTimeout)     // null
println(builder.build.waitTimeSeconds)       // null

builder.visibilityTimeout(50)  
builder.waitTimeSeconds(15)

println(builder.build.visibilityTimeout)     // 50
println(builder.build.waitTimeSeconds)       // 15
```

I used it to connect to a real SQS queue and the visibility timeout setting seemed to be working.

I restored the default values, and also updated the README which I realized I forgot to do.

Reply:
The AWS SDK javadocs have this explanation for these builder methods:

"Returns a reference to this object so that method calls can be chained together."

https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/sqs/model/ReceiveMessageRequest.Builder.html#visibilityTimeout-java.lang.Integer-

So I think the builder that it returns is the same `builder` that is originally created, and setting each property individually, rather than in one long chain, should work. It doesn't look as nice, but this is so far the easiest way I've found to set optional (in Scala terms) properties on these Java-style builders...

Reply:
Oh okay, it's mutable 🤭